source/systems/common.rst

```diff
@@ -64,5 +64,1789 @@ RDHPCS Platforms
+graphical output, **srun** can be used to run a command on the compute node;
+for example, to run a python script on a compute node that generate an image
+you can use the following method:
+
+.. code-block:: shell
+
+    srun --pty --x11 -A nesccmgmt -N 1 -t 30 python myplot.py
+
+Please also pay attention the comments in the previous section regarding X11
+forwarding.
+
+Submitting a job with arguments
+-------------------------------
+
+If you want to submit a script that accepts arguments you need to add the
+arguments after the job file name on the sbatch command. It is similar to the
+Unix method of passing arguments to a script as shown in the example below:
+
+.. code-block:: shell
+
+   sbatch batch.job arg1 arg2
+
+The command above passes arg1 as $1 and arg2 as
+$2 etc. similar to the Unix convention of argument passing.
+
+Submitting jobs with job dependencies
+-------------------------------------
+
+SLURM supports the ability to submit a job with dependencies with other jobs. A
+simple example is where job Y cannot execute until job X completes. The use of
+the -d options (``--dependency=<options>`` is the way to
+specify the job dependency.
+
+Review the sbatch manpage for a list of dependency conditions (look for
+--dependency in the sbatch options list) that can be used. Usage
+format is illustrated in the example script below that includes
+&quot;afterok&quot; as a dependency condition.
+
+Here is a simple example of how to run a chain of jobs with dependencies,
+assuming that you have a parallel helloworld.f example program in your current
+directory. Note the --parsable option that returns just the Job ID from sbatch.
+
+create/edit the file **depend** with the contents:
+
+.. code-block:: shell
+
+   #!/bin/bash
+   jid1=$(sbatch --parsable -n1 -A nesccmgmt -J sim --wrap=&quot;srun sleep 10&quot;)
+   jid2=$(sbatch --parsable -n1 -A nesccmgmt -J post --dependency=afterok:$jid1 --wrap=&quot;srun hostname&quot;)
+
+then make it executable:
+
+.. code-block:: shell
+
+   chmod 0755 depend
+
+Initiate the sequence of dependent jobs by
+
+executing **depend** from the command line:
+
+.. code-block:: shell
+
+   ./depend
+
+Notification about dependent jobs
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+By default you do not get any email notifications of your jobs. For jobs with
+dependency, it may be desirable to know if a job has been removed because the
+dependency can never be satisfied. In those cases it may be useful to submit
+dependent jobs with following notification options:
+
+.. code-block::s hell
+
+   --mail-type=END
+   --mail-user=$[mailto:USER@noaa.gov USER@noaa.gov]
+
+With these options you
+will get a notification when the job ends, and if the job is removed you will
+get an email with the following subject:
+
+
+.. code-block:: shell
+
+   Slurm Job_id=423748 Name=gfs-post Ended, Run time 00:00:00, CANCELLED, ExitCode 0
+
+
+
+How to Get Memory Usage Information
```

Comment:
The information here is not common.  Gaea does not have this command.

Reply:
removed